go.mod

```diff
@@ -5,7 +5,7 @@ go 1.17
 require (
 	github.com/ElrondNetwork/elastic-indexer-go v1.2.30
 	github.com/ElrondNetwork/elrond-go v1.3.27
-	github.com/ElrondNetwork/elrond-go-core v1.1.18
+	github.com/ElrondNetwork/elrond-go-core v1.1.21-0.20221018135819-929c00fc000d
```

Comment:
If I remember well, we should add a proper tag & a pre-release in elrond-go-core, before merging into a `rc/*`.